config/routes.rb

```diff
@@ -132,6 +132,7 @@
         put :remove_member
       end
     end
+    get 'collection/:id', controller: 'collections', action: :show_admin
```

Comment:
This overrides a route that already exists here: https://github.com/samvera/hyrax/blob/a77153b232a1950a6250e5bf152a3a2cb55d2487/config/routes.rb#L47